.github/workflows/js.yml

```diff
@@ -91,3 +91,43 @@ jobs:
 
       - name: Check types
         run: npm run typecheck
+
+  deploy-js-docs:
+    name: Deploy shirokuma API documentation
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        # This action also handles dependency caching for us
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: .nvmrc
+
+      - name: Restore from npm cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.cache_path }}
+          # Note that this caching action does NOT cache node dependencies.
+          # This is done by "actions/setup-node" instead. Here we're caching
+          # npm artifacts.
+          key: ${{ runner.os }}-test-${{ matrix.node }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Looks like the cache key doesn't make sense for that task (there is no matrix), also realized the same for the `typecheck` task